src/debugAdapter/goDebug.ts

```diff
@@ -922,6 +943,13 @@ class GoDebugSession extends DebugSession {
 					variablesReference: 0
 				};
 			} else {
+				if(v.children[0].children.length > 0){
```

Comment:
This is also a repetition since there is a ternary operator doing the same. Is there a cleaner way to do this?